CMake/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ project(test_cxx_runtime)
 
 add_executable(test_cxx_runtime typeinfo_test.cc)
 set(CMAKE_CXX_IMPLICIT_LINK_LIBRARIES "")
+if (CXX_ABI_IS_GNU)
+	add_definitions(-DCXX_ABI_IS_GNU=1)
+else ()
+	add_definitions(-DCXX_ABI_IS_GNU=0)
```

Comment:
Please can you make this path something like `CXX_ABI_IS_LLVM`?  I find this naming slightly confusing because we already support MSVC EH and GNU-flavoured Itanium EH, so having not-GNU mean LLVM-flavoured-Itanium is a bit odd.